tests/test_api.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from brownie import accounts
+from telliot_core.apps.core import TelliotCore
+from telliot_core.datafeed import DataFeed
+from telliot_core.queries.api_query import APIQuery
+from web3.datastructures import AttributeDict
+
+from telliot_feed_examples.reporters.tellorflex import PolygonReporter
+from telliot_feed_examples.sources.api_source import APIQuerySource
+
+url_test = "https://taylorswiftapi.herokuapp.com/get"
+key_str_test = "quote"
+api_feed = DataFeed(
+    source=APIQuerySource(url=url_test, key_str=key_str_test),
+    query=APIQuery(url=url_test, key_str=key_str_test),
+)
+
+
+@pytest.mark.skip
```

Comment:
Unskip this, put the test file in `tests/reporter/`, and rename the file to `test_report_api_query.py`